model/project_ref.go

```diff
@@ -114,6 +114,9 @@ type ProjectRef struct {
 	ContainerSizeDefinitions []ContainerResources `bson:"container_size_definitions,omitempty" json:"container_size_definitions,omitempty" yaml:"container_size_definitions,omitempty"`
 	ContainerSecrets         []ContainerSecret    `bson:"container_secrets,omitempty" json:"container_secrets,omitempty" yaml:"container_secrets,omitempty"`
 
+	// Filters settings
+	ParsleyFilters []ParsleyFilter `bson:"parsley_filters,omitempty" json:"parsley_filters,omitempty"`
```

Comment:
Preference to move this to the bottom since those are also about UI views

Reply:
I'll wait until this is merged in and follow suit for #6516 👍